docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Building the Documentation
+
+Dioxus uses a fork of MdBook with multilanguage support. To build the documentation, you will need to install the forked version of MdBook.
+
+```sh
+cargo install mdbook --git https://github.com/Demonthos/mdBook.git --branch master
+```
+
+Then, you can build the documentation by running:
+
+Linux and MacOS:
+
+```sh
+cd docs && cd guide && mdbook build -d ../nightly/guide && cd .. && cd router && mdbook build -d ../nightly/router && cd .. && cd ..
+```
+
+Windows:
+
+```cmd
+cd docs; cd guide; mdbook build -d ../nightly/guide; cd ..; cd router; mdbook build -d ../nightly/router; cd ..; cd ..
```

Comment:
I think all 3 systems use the Linux/MacOS command. The Windows version listed here doesn't work on my Windows 10 machine, but the other one does.

Reply:
This is based on the terminal not the operating system. I updated this to have every command separately which should work on all terminals